docs/source/technical_tutorials/access/external_access_from_hub.ipynb

```diff
@@ -0,0 +1,1310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>Add a comment about always optimizing the cache, and link to the Cloud Native guide on this topic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/535/discussion/#comment-2600614984'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>